Src/library/base/elks/kernel/type.e

```diff
@@ -1,12 +1,13 @@
 ﻿note
 	description: "Representation of an Eiffel type."
+	remark: "At any given time, there is no more than one instance of TYPE representing a given Eiffel type."
```

Comment:
I do not see how it can work at the moment in SCOOP if two objects belong to different regions. If this were indeed the case, the type of `{ANY}.generating_type` would be separate. Right now this remark seems to be misleading and implementation that supports this in SCOOP break its safety guarantees.
I'd suggest to either review the whole concept behind the single-type-object idea to make it sound, useable and practical in the first place, or remove the remark for the time being.